pyiron_base/project/generic.py

```diff
@@ -1036,6 +1065,41 @@ def refresh_job_status_based_on_job_id(self, job_id, que_mode=True):
                 if not self.queue_check_job_is_waiting_or_running(job):
                     self.db.set_job_status(job_id=job_id, status="aborted")
 
+    def _refresh_job_status_file_table(self, df):
+        """
+        Internal function to refresh the job table and update the job table with the status from the queuing system.
+
+        Args:
+            df (pandas.DataFrame): job table from the file based database
+
+        Returns:
+            pandas.DataFrame: updated job table with status from the queuing system
+        """
+
+        def convert_queue_status(queue_status):
+            convert_status_dict = {"pending": "submitted"}
+            if queue_status in convert_status_dict.keys():
+                return convert_status_dict[queue_status]
+            else:
+                return queue_status
```

Comment:
Isn't this just 

```python
if queue_status == "pending":
  return "submitted"
else:
  return queue_status
```

If you want to have a dict in case there will be more status in the future to map
```python
return {"pending": "submitted"}.get(queue_status, default=queue_status)
```
is still easier to read.